### PR TITLE
blueprint ⇢ API description

### PR DESCRIPTION
**Pull request by honzajavorek**
Generalising mentions of blueprint to mentions of API description.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -6,10 +6,10 @@
 [![Coverage Status](https://coveralls.io/repos/apiaryio/dredd/badge.svg?branch=master)](https://coveralls.io/r/apiaryio/dredd?branch=master)
 [![Join the chat at https://gitter.im/apiaryio/dredd](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/apiaryio/dredd?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-![Dredd API Blueprint testing tool](https://raw.github.com/apiaryio/dredd/master/img/dredd.png?v=3&raw=true)
+![Dredd - HTTP API Testing Framework](https://raw.github.com/apiaryio/dredd/master/img/dredd.png?v=3&raw=true)
```
Comment (pksunkara) [inline docs/index.md]: API Description Testing Framework

File: docs/overview.md [hunk 2]
```diff
@@ -65,27 +65,27 @@ When sending test reports to Apiary, Dredd inspects the environment where it was
...
+## Testing API description
 
 ### Documentation Testability
 
-API Blueprint allows usage of URI templates. If you want to have API documentation to be complete and testable, do not forget to describe all URI-used parameters and provide examples to make Dredd able to expand URI templates with given example values.
+API Blueprint allows usage of URI templates. If you want to have API description to be complete and testable, do not forget to describe all URI-used parameters and provide examples to make Dredd able to expand URI templates with given example values.
 
 ### Isolation
 
 API Blueprint structure conforms to the REST paradigm, so in the API Blueprint are documented Resources and their Actions.
```
Comment (pksunkara) [inline docs/overview.md]: API Blueprint?

File: docs/overview.md [hunk 3]
```diff
@@ -65,27 +65,27 @@ When sending test reports to Apiary, Dredd inspects the environment where it was
...
 
-It's very likely that your blueprint will not be testable as-is, because actions in the reference will not be sorted in proper order for API's application logic workflow.
+It's very likely that your API Blueprint document will not be testable as-is, because actions in the reference will not be sorted in proper order for API's application logic workflow.
```
Comment (pksunkara) [inline docs/overview.md]: Blueprint?

**Comment by pksunkara** [comment]
Looks good except my comments.

File: docs/index.md (hunk 1 above)
Comment (honzajavorek) [inline docs/index.md]: The semi-official claim right now is "HTTP API Testing Framework" for a longer time and it was already in the README. Not sure why here it was as "API Blueprint testing tool".

File: docs/overview.md (hunk 2 above)
Comment (honzajavorek) [inline docs/overview.md]: Yup, some places are still left intact. E.g. here the sentence says "API Blueprint structure conforms to the REST paradigm", which isn't valid to OAI in my opinion. Rephrasing such sections should be part of "Let's add Swagger docs" PR.

File: docs/overview.md (hunk 3 above)
Comment (honzajavorek) [inline docs/overview.md]: The same: https://github.com/apiaryio/dredd/pull/492#discussion_r64224991

**Comment by honzajavorek** [comment]
Merging, feel free to continue commenting ⇢ I can provide fixes in subsequent PRs if necessary.

**Pull request merged**